visual_behavior/data_access/loading.py

```diff
@@ -185,7 +185,12 @@ def get_platform_paper_experiment_table():
 
     # add cell type columm
     experiment_table = utilities.add_cell_type(experiment_table)
-    experiment_table.cell_type.unique()
+    # add other useful columns
+    experiment_table = utilities.add_n_relative_to_novel_1(experiment_table)
+    experiment_table = utilities.add_last_familiar(experiment_table)
+    experiment_table = utilities.add_last_familiar_active(experiment_table)
+    experiment_table = utilities.add_first_novel(experiment_table)
```

Comment:
lets change the column first_novel to first_post_novel.

Reply:
how about 'first_novel_repeat' or 'second_novel' instead?

Reply:
or even more explicit, "first_passing_novel_repeat" or "second_passing_novel", just so we don't confuse absolute order (the true experience of the mouse) with the order of sessions that are available for analysis (only the passing ones)